### ISSUE TITLE
Undo thread rating

### ISSUE THREAD

**Issue by Eldenroot** [Darth-Apple/Average-Profile-Ratings#6]
Add undo thread rating (when you change your mind) and you would like to change the rating :)

**Comment by Darth-Apple** [comment]
I'm surprised this isn't in MyBB core. I'll probably implement something for this in a future release! :)

**Comment by Eldenroot** [comment]
There is a plugin for that but I do not want to use XXX plugins, better to have all in one :)

**Comment by Darth-Apple** [comment]
I agree. I went ahead and released the current version so that it would be available for the MyBB community. There are a number of very good ideas to make the ratings system more useful, so I'm looking into it. Being able to see your rating in addition to the thread rating is also a very good suggestion.
